### ISSUE TITLE
Add hook to Nextflow configs generated by VDSL3

### ISSUE THREAD

**Issue by rcannood** [viash-io/viash#415]
```groovy
platforms:
  - type: nextflow
    config:
      labels:
        lowmem: ...
        midmem: ...
      include_in_config: |
        includeConfig(...)
      include: /workflows/infrastructure.config
```

```groovy
platforms:
  - type: nextflow
    config:
      labels:
        lowmem: ...
        midmem: ...
      include:
        - /path/to/config.config
        - |
          profiles {
            ...
          }
```

```groovy
platforms:
  - type: nextflow
    config:
      labels:
        lowmem: ...
        midmem: ...
      include:
        - path: /path/to/config.config
        - text: |
          profiles {
            ...
          }
```

```groovy
functionality:
  resource: /path/to/config.config
platforms:
  - type: nextflow
    config:
      labels:
        lowmem: ...
        midmem: ...
      include: |
        includeConfig("config.config")
```

* Move labels to config object
* Keep order of items

**Comment by rcannood** [comment]
Proposed format:

Let `include` be a OneOrMore[String]


```groovy
platforms:
  - type: nextflow
    config:
      labels:
        lowmem: ...
        midmem: ...
      include:
        - |
          profiles {
            ...
          }
```

**Comment by Grifs** [comment]
Implemented in Viash 0.7.4
